core/vil/vil_pyramid_image_view.hxx

```diff
@@ -30,9 +30,8 @@ vil_pyramid_image_view<T>::vil_pyramid_image_view(vil_image_view_base_sptr image
   if (!img)
     return;
 
-  // it works with grey scale images
-  if (img->nplanes() != 1)
-    return;
+  //if (img->nplanes() != 1)
+   // return;
```

Comment:
Remove unnecessary code unless there is some specific reason it's still useful as a comment